src/layout/msWorkspace/msWorkspace.ts

```diff
@@ -142,7 +142,15 @@ export class MsWorkspace extends WithSignals {
                         width: msWindowData.width,
                         height: msWindowData.height,
                     },
-                    matchingInfo
+                    matchingInfo,
+                    // Set yesterday for older state because we don't want windows to be recognised as just created
```

Comment:
```suggestion
                    // Set createdAt to yesterday when migrating older state, because we don't want restored old windows to be treated as just having been created
```